site/docs/pages/index.mdx

```diff
@@ -7,25 +7,49 @@ layout: landing
 showOutline: false
 ---
 
-import { base } from 'viem/chains';
-import { OnchainKitProvider } from '@coinbase/onchainkit';
-import { Address, Avatar, Badge, Name, Identity } from '@coinbase/onchainkit/identity';
-import { Swap, SwapAmountInput, SwapButton, SwapMessage, SwapToggleButton } from '@coinbase/onchainkit/swap';
-import { Wallet, ConnectWallet } from '@coinbase/onchainkit/wallet';
-import App from '../components/App';
-import FarcasterCastCard from '../components/FarcasterCastCard';
-import SwapWrapper from '../components/SwapWrapper';
-import TokenSelector from '../components/TokenSelector';
-import { WalletDemo } from '../components/WalletDemo';
+import { base } from "viem/chains";
+import { OnchainKitProvider } from "@coinbase/onchainkit";
+import {
+  Address,
+  Avatar,
+  Badge,
+  Name,
+  Identity,
+} from "@coinbase/onchainkit/identity";
+import {
+  Swap,
+  SwapAmountInput,
+  SwapButton,
+  SwapMessage,
+  SwapToggleButton,
+} from "@coinbase/onchainkit/swap";
+import { Wallet, ConnectWallet } from "@coinbase/onchainkit/wallet";
+import App from "../components/App";
+import FarcasterCastCard from "../components/FarcasterCastCard";
+import SwapWrapper from "../components/SwapWrapper";
+import TokenSelector from "../components/TokenSelector";
+import { WalletDemo } from "../components/WalletDemo";
```

Comment:
Make sure to rebase from Origin. Made a few improvements with the homepage early today.